### ISSUE TITLE
possible to have remove_all in atomic metadata operation?

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods#7871]
## Feature
Would be possible to have "remove all available metadata on an object without knowing attributes and values" in one shot?

**Comment by korydraughn** [comment]
Yes, it's possible. We'd have to make a decision about how to expose the functionality. There are three ways to do that.
- Modify the atomic metadata API
- Modify a different, existing API
- Create a new API endpoint

Modifying the atomic metadata API seems to be the most approach, so we can start there.

**Comment by mstfdkmn** [comment]
Thanks. One of the motivations in requesting this is that the atomic metadata api has two options for the operation (`add` and `remove`), it doesn't contain `set`. So, in case a need to overwrite an existing metadata pair, we need to beforehand know attribute and value. 
And the other reason will be that with any client which can call the atomic end point we can clear all existing metadata and add new metadata in one call easily.

**Comment by korydraughn** [comment]
That's correct. The exclusion of `set` is intended because it could be implemented in terms of `add` and `remove`.

We'll look into getting this implemented in an efficient manner.
